pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/FunctionsImpl.java

```diff
@@ -226,4 +226,4 @@ public static void mergeJson(String json, Builder builder) throws IOException {
     public static String printJson(MessageOrBuilder msg) throws IOException {
         return JsonFormat.printer().print(msg);
     }
-}
+}
```

Comment:
nit sure about this change

Reply:
this PR was also having changes from previous PR so, rebase might have changed last line spaces.

Reply:
@sijie I have added `PulsarSinkTets` . However, I feel keeping Kinesis-Connector e2e test is not harmful as it's just a test and dependency-scope is also test. I think it is helping to add more test code-coverage in pulsar-function where existing testcases are not touching some of the code paths. We can definitely remove it in future and only keep PulsarSinkTest if it is causing any issue for dependency management but until we can keep it as it will be helpful for future development testing.

Reply:
removed dependency and test.